taf/testlib/virtual_env.py

```diff
@@ -28,7 +28,9 @@
 #  "instance_type": "settings",
 #  "id": "993",
 #  "images_share_path": "/mnt/berta/oses/openstack",
-#  "mgmt_ip_cidr": "11.212.23.11/8",
+#  "external_net_ip_cidr": "11.212.23.0/24",
+#  "external_net_pool_start": "100",
+#  "external_net_pool_end": "199",
```

Comment:
192.168 is okay or other RFC 1918 addresses are okay as long they don't match actual internal subnets.